src/assets/scss/helpers.scss

```diff
@@ -89,6 +88,10 @@
   -moz-user-select: none;
   -ms-user-select: none;
   white-space: nowrap;
+  // ficxing button spacing issue. Fix- center aligning the buttons
+  display: inline-flex !important;
```

Comment:
You mentioned that you're using `!important` here to override the displays that we've defined in some buttons. 

I was thinking about it, and Instead of `!important` let's just remove the display definition from those other places. We should keep the rules in here as general as possible and should allow other places to be able to override them.